packages/validation/src/nodebalancers.schema.ts

```diff
@@ -80,12 +106,19 @@ export const createNodeBalancerConfigSchema = object({
 
 export const UpdateNodeBalancerConfigSchema = object({
   algorithm: mixed().oneOf(['roundrobin', 'leastconn', 'source']),
-  check_attempts: number(),
+  check_attempts: number()
+    .min(CHECK_ATTEMPTS.MIN)
+    .max(CHECK_ATTEMPTS.MAX)
+    .integer(),
   check_body: string().when('check', {
     is: 'http_body',
     then: string().required('An HTTP body regex is required.'),
   }),
-  check_interval: number().typeError('Check interval must be a number.'),
+  check_interval: number()
+    .min(CHECK_INTERVAL.MIN)
+    .max(CHECK_INTERVAL.MAX)
+    .typeError('Check interval must be a number.')
```

Comment:
```suggestion
```suggestion
    .min(
      CHECK_INTERVAL.MIN,
      `Interval should be greater than or equal to ${CHECK_INTERVAL.MIN}.`
    )
    .max(
      CHECK_INTERVAL.MAX,
      `Interval should be less than or equal to ${CHECK_INTERVAL.MAX}.`
    )
    .typeError('Interval must be a number.')
```
```